include/verilated_funcs.h

```diff
@@ -2083,6 +2104,97 @@ static inline void VL_ASSIGNSEL_WW(int rbits, int obits, int lsb, WDataOutP iowp
     _vl_insert_WW(iowp, rwp, lsb + obits - 1, lsb, rbits);
 }
 
+//====================================================
+// Range assignments
+
+// These additional functions copy bits range [obis+roffset-1:roffset] from rhs to lower bits
+// of lhs. Rhs should always be wider than lhs.
+static inline void VL_ASSIGNSEL_II(int rbits, int obits, CData& lhsr, IData rhs,
+                                   int roffset) VL_PURE {
```

Comment:
There's already VL_ASSIGNSEL_II, so if this needs to be different need new name.

Reply:
Can we see it as a function overloading? Original batch of functions has `lsb` as the third argument to specify offset bits of LHS, this new batch of functions remove `lsb` argument and add `roffset` argument at last to specify offset bits of RHS.

Reply:
Relying on function overloading as a concept is fine, my objection is it this is doing a different operation, so the function name needs to be different to avoid reader confusion.

Reply:
Renamed to `VL_SELASSIGN_XX` to shows it  does some reverse operation of `VL_ASSIGNSLE_XX` functions.